src/crimson/os/seastore/onode_manager/staged-fltree/fltree_onode_manager.cc

```diff
@@ -120,6 +120,7 @@ FLTreeOnodeManager::erase_onode_ret FLTreeOnodeManager::erase_onode(
   OnodeRef &onode)
 {
   auto &flonode = static_cast<FLTreeOnode&>(*onode);
+  flonode.populate_recorder(trans);
```

Comment:
Is this more accurate?
```c++
assert(flonode.is_alive());
if (flonode.status == FLTreeOnode::status_t::MUTATED) {
  flonode.populate_recorder(trans);
}
```

Reply:
And looks unit test is failed because of not checking the status like above.

Reply:
yeah, will correct it.

Reply:
Done.